File: seed/pydantic-v2/trace/resources/v_2/resources/v_3/resources/problem/types/void_function_signature_that_takes_actual_result.py
from pydantic import BaseModel
from typing import List
from resources.v_2.resources.v_3.resources.problem.types.parameter import Parameter
from resources.commons.types.variable_type import VariableType
from dt import datetime
from core.datetime_utils import serialize_datetime
class VoidFunctionSignatureThatTakesActualResult(BaseModel):
    parameters: List[Parameter]
    actual_result_type: VariableType = 
    class Config:
        frozen = True
        smart_union = True
        json_encoders = {datetime: serialize_datetime}

File: seed/ruby-sdk-v2/trace/lib/seed/v_2/problem/void_function_signature_that_takes_actual_result.rb

module Seed
    module Types
        class VoidFunctionSignatureThatTakesActualResult < Internal::Types::Model
            field :parameters, Internal::Types::Array[Seed::v_2::problem::Parameter], optional: false, nullable: false
            field :actual_result_type, Seed::commons::VariableType, optional: false, nullable: false
        end
    end
end
